code/__HELPERS/roundend.dm

```diff
@@ -414,7 +416,7 @@
 		if(!A.members)
 			continue
 		all_teams |= A
-	
+
```

Comment:
As per new review guidelines i am required to have a deep review comment on each review i make,as such,i decided to do an interview with a certain piece of code

Now playing: The code interview.

Staring `Random line spacing change` and `alexkar598` as the interviewer! 
___
alexkar598:
>Hello random line spacing change,you are you doing?
___
Random line spacing change: 
> I'm doing fine,thank you!
___
alexkar598:
>You have been doing what here?
___
Random line spacing change: 
>Oh nothing! i just clutter up the git objects being virtually useless and also messing with people trying to develop things
___
alexkar598:
>But how will that affect anyone? Only KMC codes for st-ds-1561-3546-fesklah-fesu-69 and we all know it
___
Random line spacing change: 
>isn't the new shorthand for deepspace 13 74c865e8d169901bdd4ef43e5ed11af45405e845?
___
alexkar598:
>No thats just your commit sha hash
___
Random line spacing change: 
>Oh...I was wondering,is there a character limit to github reviews?
___
alexkar598:
>I suppose there isn't any!Anyways, have you been playing ss13 recently?
___
Random line spacing change: 
>Nope,BYOND is just always down,it seems lummox is storing the server in a broom closet and the mop dripped space goo on it
___
alexkar598:
>Welp,this is all the time i have for today,see you later random line spacing change!
___
Random line spacing change: 
>See you later